File: javascript/media_idades.js
let idade = parseInt(prompt("Digite a idade"));

let somaIdades = 0, qtdIdades = 0;

if (idade < 0) {
    console.log("Não é possivel calcular");
} else {
    somaIdades += idade;

    while(idade > 0) {
        idade = parseInt(prompt("Digite a idade"));
        somaIdades += idade;
        qtdIdades++;
    }

    somaIdades -= idade;
    const mediaIdades = somaIdades / qtdIdades;

    console.log("Média = %.2f", mediaIdades);
}

File: python/media_idades.py
soma_idades = 0
quantidade = 0

print("Digite as idades:")
idade = int(input())

if idade < 0:
    print("IMPOSSIVEL CALCULAR")
else:
    soma_idades += idade
    quantidade += 1

    while idade > 0:
        idade = int(input())
        soma_idades += idade
        quantidade += 1

    quantidade -= 1
    soma_idades -= idade
    media = soma_idades / quantidade
    print(f"MEDIA = {media:.2f}")
